README.md

```diff
@@ -2,6 +2,122 @@
+
+---
+
+### Task estimation
+
+**What's a task ?**
+
+A task is defined by "a piece of work to be done or undertaken". So, in software progamming for example, a task could be a feature, a code review as well as writting documentation or resolving an issue.
+
+**How to estimate a task ?**
+
+As seen before with project estimation, several steps are needed to correctly and efficiently estimate a task duration. Some of these are also applicable to the task scope.
+We will need to estimate not only the time we will need to accomplish the task but also many other things like human resources needed to handle the task, infrastructure cost, meetings needed to follow the task advancement, code reviews and many more.
+
+**Task estimation cons and pros**
+
+When talking about project or task estimation we obviously think about how the estimations will help the team to work efficiently and gain time on a project. But most of the time we don't take into account that taking too much time on estimations is also a lost of time in the sense that estimating could save us time thanks to a better organization but it's only worth if it's frequently done with short estimation and relatively quick decisions.
+
+[Read more about project and task estimation](https://medium.com/globalluxsoft/time-estimation-in-software-development-a4a495c8eb6c)
+
+### SCRUM method
+
+**What's SCRUM method ?**
+
+Scrum is an agile method dedicated to "project management". This management method, or rather this Project Management Framework, aims to improve the productivity of your team.
+This method includes several roles that the whole team must play and it's very helpful to the task estimation.
+
+### SCRUM method roles
+
+**SCRUM Master**
+
+The Scrum Master is first and foremost a member of the project team, he is not a superior but rather a coach, these main tasks are listed below:
+
+- Ensures that Scrum's principles and values are respected.
+- Facilitates communication within the team.
+- Seeks to improve the productivity and know-how of his team.
+
+**The team**
+
+The team is the heart of the SCRUM method, without a team, no work done, it seems logical. They do not have defined roles, it will mostly depend on the type of project. 
+Here are the main characteristics of a team:
+
+- No definite role: architect, developer, tester.
+- All team members bring their know-how to accomplish the tasks.
+- Sizes from 6 to 10 people in general and up to 200 people in particular.
+
+**Product Owner**
+
+Within a Scrum team, the Product Owner makes the link between the business part and the technical part of the project, here are his main objectives: 
+
+- Business expert, defines functional specifications
+- Prioritizes the features to be developed or corrected
+- Validates the developed functionalities
+- Plays the role of the customer
+
+### Sprints
+
+The Scrum life cycle is punctuated by iterations of a few weeks, the sprints.
```

Comment:
Thanks to our Business Modeling lecture, you could specify the time of sprint ? (between 5 days to 2 weeks in general)